### PR TITLE
Updated readme because of new functionality

### PR DESCRIPTION
**Pull request by BTWS2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +44,24 @@ The first line of the newly created file will contain the link to the selected e
...
+|       <img src="assets/refresh.svg" alt="refresh" width=15px;/>        | Reload the list of activities.                                                                                                                                                        |
+|  <img src="assets/notification.svg" alt="notification" width=15px;/>   | Notifies users in case of unread feedback. The icon links to https://dodona.ugent.be/notifications.                                                                                   |
+| <img src="assets/content-page-read.svg" alt="checkmark" width=15px;/>  | Submit solution to Dodona. Alternative for `Alt+D` and `Ctrl+Shift+P`, `Dodona: Submit Solution`.                                                                                     |
+|          <img src="assets/edit.svg" alt="edit" width=15px;/>           | Opens the description of an exercise series or course in a new browser window. (Hover with your cursor above an exercise series or course in the left column with Dodona activities.) |
```
Comment (stijndcl) [inline README.md:63]: Use commas when summing up, hover `over`
```suggestion
|          <img src="assets/edit.svg" alt="edit" width=15px;/>           | Opens the description of an exercise, series or course in a new browser window. (Hover with your cursor over an exercise, series or course in the left column with Dodona activities.) |
```

File: README.md [hunk 2]
```diff
@@ -44,11 +44,24 @@ The first line of the newly created file will contain the link to the selected e
...
+| <img src="assets/open-description.svg" alt="description" width=15px;/> | Show the exercise description in VS Code. (Hover with your cursor above an exercise in the left column with Dodona activities.)                                                       |
```
Comment (stijndcl) [inline README.md:64]:
```suggestion
| <img src="assets/open-description.svg" alt="description" width=15px;/> | Show the exercise description in VS Code. (Hover with your cursor over an exercise in the left column with Dodona activities.)                                                       |
```

**Review by stijndcl — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BTWS2) [reply README.md:63]: I meant "oefeningenreeks" in Dutch. Does this translate to `exercise series`?

Comment (stijndcl) [reply README.md:63]: Oh, okay. I'll edit my suggestion.

In that case: both Dodona itself and our plugin refer to these as `series` everywhere, so for the sake of consistency I feel like it's better to stick with that. If you look at the URLs you'll also notice this.

`dodona.ugent.be/series/XXX`, `dodona.ugent.be/courses/XXX/series`, etc.

File: README.md [hunk 3]
```diff
@@ -17,10 +17,10 @@ You can authenticate by creating an API token. Using such token, VS Code can sub
 
 1. <img src="assets/sideview.png" alt="cogwheel" width=30px;/> Dodona icon in the Activity Bar (icon in left column)
 2. `Open Settings`
-   ![API token](assets/api-token2.png)
+   ![Open settings](assets/open-settings.png)
 3. Paste the API token you just made in text box.
 4. In the left column you get an overview of the exercise series of the course (expand it if necessary).
```
Comment (BTWS2) [inline README.md:22]: Can you change this to ` In the left column you get an overview of the exercises and series of the course (expand it if necessary).`?

File: README.md (hunk 1 above)
Comment (BTWS2) [reply README.md:63]: I searched for it here https://dodona-edu.github.io/en/guides/teachers/exercise-series-management/ but there was no English translation available of that page. Thank you for your clarification.

File: README.md (hunk 3 above)
Comment (stijndcl) [reply README.md:22]: Like this?
```suggestion
4. In the left column you get an overview of the exercises and series of the course (expand it if necessary).
```

Comment (BTWS2) [reply README.md:22]: Yes, thank you.

**Review by stijndcl — APPROVED** [review]

**Review by thepieterdc — APPROVED** [review]

**Pull request merged**